tests/test_mesosphere_signer.py

```diff
@@ -76,16 +76,22 @@ def test_medianize():
 
 
 def test_create_signer_instance():
-    cfg = get_configs([])
-    signer = TellorSigner(cfg)
-
-    assert (
-        signer.cfg.address == 0xACC2D27400029904919EA54FFC0B18BF07C57875
-    )  # tellor contract
-    assert signer.w3.isConnected() == True
-    assert signer.secret_test == "passed"
-    assert signer.acc.address != None
-    assert signer.acc.address != ""
+
+    def create_signer_on_network(network, contract_address):
+        cfg = get_configs(["-n", network])
+        signer = TellorSigner(cfg)
+        network = cfg["network"]
+
+        assert (
+            signer.cfg.address[network] == contract_address
+        )  # tellor contract
+        assert signer.w3.isConnected() == True
```

Comment:
This one fails when I run the tests

Reply:
yeah same for me. POKT I think is rejecting that RPC call